content/practices/high-contrast/high-contrast-practice.html

```diff
@@ -0,0 +1,779 @@
+
+        <table class="data">
+          <caption>Button Switch Example with Selected High Contrast User Settings in Windows 11</caption>
+          <thead>
+            <tr>
+              <th>Setting</th>
+              <th>Screen Shot</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>Contrast Theme: none (default)</td>
+              <td><img src="./images/currentcolor-windows11-theme-none.png" alt="Screen shot of switch example with no contrast theme applied" style="height: 90px"></td>
+            </tr>
+            <tr>
+              <td>Contrast Theme: Night sky</td>
+              <td><img src="./images/currentcolor-windows11-theme-night-sky.png" alt="Screen shot of switch example with night sky contrast theme applied" style="height: 87px"></td>
+            </tr>
+            <tr>
+              <td>Contrast Theme: Desert</td>
+              <td><img src="./images/currentcolor-windows11-theme-desert.png" alt="Screen shot of switch example with desert contrast theme applied" style="height: 95px"></td>
+            </tr>
+          </tbody>
+        </table>
+
+        <h3 id="currentcolor-examples">Examples using <code>currentcolor</code> keyword</h3>
+        <ul>
+          <li><a href="../../patterns/checkbox/examples/checkbox-mixed.html">Checkbox (Mixed-State)</a></li>
+          <li><a href="../../patterns/disclosure/examples/disclosure-faq.html">Disclosure (Show/Hide) for Answers to Frequently Asked Questions</a></li>
+          <li><a href="../../patterns/radio/examples/radio-activedescendant.html">Radio Group  Using aria-activedescendant</a></li>
+          <li><a href="../../patterns/slider/examples/slider-seek.html">Media Seek Slider</a></li>
+          <li><a href="../../patterns/slider/examples/slider-temperature.html">Vertical Temperature Slider</a></li>
+        </ul>
+      </section>
+
+      <section id="svg">
+        <h2>SVG Graphics versus Bit-Mapped Images for Components</h2>
+
+        <h3>Limitations of Bit-Mapped Images</h3>
+
+        <p>The colors of pixels used in bit-mapped images (e.g. .png, .jpeg) do not respond to changes in media query values making them difficult or impossible for people with some types of visual impairments to see when the change contrast settings.  Low resolution images also do not scale smoothly when the browser zoom features are used to increase the size of rendered content and the resulting distortion will make it more difficult or impossible for people to identify the component.</p>
```

Comment:
The first sentence isn’t entirely true anymore since authors can manipulate the appearance of bitmap images with CSS techniques like `filter: contrast()`, `mix-blend-mode`, etc. in reaction to a user preference media query.

Reply:
Ok, it should acknowledge this technique.   I will need to research the capabilities of the technique.

Reply:
@adampage 
I made some changes to the bit-mapped image section based on your feedback, could you review an comment?

Reply:
Thanks @jongund, these changes look great. 👍🏻